src/thinblock.cpp

```diff
@@ -369,7 +369,7 @@ bool CXThinBlock::HandleMessage(CDataStream &vRecv, CNode *pfrom, string strComm
         if (pIndex->nChainWork <= chainActive.Tip()->nChainWork)
         {
             vector<CInv> vGetData;
-            vGetData.push_back(CInv(MSG_THINBLOCK, inv.hash));
+            vGetData.push_back(inv);
```

Comment:
I agree there's not much benefit here to request a thinblock, but in the future i think we should consider relaxing this and request only when pIndex->nChainWork < chainActive.Tip()->nChainWork and still request any orphans using a get_xthin